codeql/windows-drivers/queries/Windows/wdk/kmdf/DeviceInitApi/DeviceInitApi.ql

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @name Access of WDF init API after WdfDeviceCreate
+ * @description Calling a WDF init API on a device init object after calling WdfDeviceCreate can cause system instability.

```

Comment:
Added details about why this causes instability (the framework takes ownership of the object).  This is not a security bug and as such is tagged correctness rather than security.